.github/workflows/cypress.yml

```diff
@@ -0,0 +1,35 @@
+name: Cypress
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - stable*
+
+env:
+  APP_NAME: nextcloud-vue
+
+jobs:
+  jest:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: ['12.x']
+
+    name: Runner ${{ matrix.containers }}
```

Comment:
No parallelism anymore (Wasn't needed because we have 1 test, but how would we add more?)

Reply:
I don't think we gain much here. The whole `Cypress` action runs for 4:00 min. The cypress run took 35 s, of which running the actual AppSideber.visual.js test took 10 s. We would need to add a lot of tests to really save time.